src/runtime/tests/manifest-test.ts

```diff
@@ -3764,15 +3765,76 @@ recipe Three
     assert.equal(multiParam.params['foo'], 'Text');
     assert.equal(multiParam.params['bar'], 'Number');
     assert.equal(multiParam.params['baz'], 'Boolean');
-    assert.deepEqual(multiParam.targets, ['Store', 'Handle', 'HandleConnection', 'Schema', 'SchemaField']);
+    assert.deepEqual(multiParam.targets, ['Store', 'Handle', 'HandleConnection', 'Schema', 'SchemaField', 'Recipe']);
     assert.equal(multiParam.retention, 'Runtime');
 
     const goodForAll = manifest.annotations['goodForAll'];
     assert.isEmpty(goodForAll.params);
     assert.isEmpty(goodForAll.targets);
     assert.equal(goodForAll.retention, 'Runtime');
 
-    // TODO: store and serialize manifest items' annotations
-    // assert.equal(manifest.toString(), manifestStr);
+    assert.equal(manifest.toString(), manifestStr);
+
+    for (const recipe of manifest.recipes) {
+      const cloneRecipe = recipe.clone();
+      assert.equal(cloneRecipe.toString(), recipe.toString());
+    }
+  });
+  it('verifies referencing annotations', async () => {
+    await assertThrowsAsync(async () => await Manifest.parse(`
+        @nonexistent()
```

Comment:
Is it possible to have annotations with no parens?

```
@something
recipe Blah
```

Reply:
yes, i added todos for this (with a bug number!). this is temporarily needed until I can deprecate all existing annotations (and i need the support for canonical annotations to do that). 
long story there is an `@active` annotation on a recipe in arc-serializer and this is the least painful workaround to distinguish between them that i could think of.

Reply:
Thanks for explaining, SGTM!